### PR TITLE
Modify storage snapshottable test for Windows

### PR DESCRIPTION
**Pull request by jingxu97**
Update snapshottable test for Windows

Change-Id: I9d6beb25078c1941e405bf61545caae30326ddd1

**What type of PR is this?**

**What this PR does / why we need it**:

**Which issue(s) this PR fixes**:

Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

**Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.**:

```docs

```

### CODE REVIEW

**Comment by jingxu97** [comment]
assign @msau42

File: test/e2e/framework/exec_util.go [hunk 1]
```diff
@@ -23,7 +23,7 @@ import (
 	"net/url"
 	"strings"
 
-	"k8s.io/api/core/v1"
+	v1 "k8s.io/api/core/v1"
```
Comment (oomichi) [inline test/e2e/framework/exec_util.go:26]: This seems unnecessary change in this PR.

File: test/e2e/storage/testsuites/snapshottable.go [hunk 2]
```diff
@@ -274,8 +276,7 @@ func (s *snapshottableTestSuite) DefineTests(driver TestDriver, pattern testpatt
 					StopPod(cs, restoredPod)
 				})
 				framework.ExpectNoError(e2epod.WaitForPodRunningInNamespaceSlow(cs, restoredPod.Name, restoredPod.Namespace))
-
-				command = "cat /mnt/test/data"
+				e2evolume.GenerateReadFileCmd(datapath)
```
Comment (oomichi) [inline test/e2e/storage/testsuites/snapshottable.go:279]: I guess here should be
```
command = e2evolume.GenerateReadFileCmd(datapath)
```

**Review by oomichi — COMMENTED** [review]
/cc @oomichi

File: test/e2e/storage/testsuites/snapshottable.go (hunk 2 above)
Comment (jingxu97) [reply test/e2e/storage/testsuites/snapshottable.go]: updated, thanks!

File: test/e2e/framework/exec_util.go (hunk 1 above)
Comment (jingxu97) [reply test/e2e/framework/exec_util.go:26]: this was automatically updated by my code editor. so I am ok either remove it or leave it like this. What do you think?

**Comment by jingxu97** [comment]
cc @mattcary @saikat-royc @msau42

**Comment by Jiawei0227** [comment]
I think it lgtm. 
/lgtm

File: test/e2e/framework/exec_util.go (hunk 1 above)
Comment (xing-yang) [reply test/e2e/framework/exec_util.go:26]: Can you remove this?

Comment (jingxu97) [reply test/e2e/framework/exec_util.go]: done.
